packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx

```diff
@@ -195,7 +188,7 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
       },
       borderless && {
         borderColor: 'transparent',
```

Comment:
Is `borderColor` needed here now that `border: none` is set?

Reply:
Removed 👍